squidpy/datasets/__init__.py

```diff
@@ -1,2 +1,3 @@
 from squidpy.datasets._image import *  # noqa: F403
 from squidpy.datasets._dataset import *  # noqa: F403
+from squidpy.datasets._10x_datasets import visium
```

Comment:
add type stubs as for the other datasets

Reply:
Would say let's postpone, afaik it was needed for others because we compile the functions. If needed, will add in the future.